staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -545,6 +551,113 @@ message ListPodSandboxResponse {
     repeated PodSandbox items = 1;
 }
 
+message PodSandboxStatsRequest {
+    // ID of the pod for which to retrieve stats.
+    string pod_sandbox_id = 1;
+}
+
+message PodSandboxStatsResponse {
+    PodSandboxStats stats = 1;
+}
+
+// PodSandboxStatsFilter is used to filter containers.
+// All those fields are combined with 'AND'
+message PodSandboxStatsFilter {
+    // ID of the container.
+    string id = 1;
+    // LabelSelector to select matches.
+    // Only api.MatchLabels is supported for now and the requirements
+    // are ANDed. MatchExpressions is not supported yet.
+    map<string, string> label_selector = 2;
+}
+
+message ListPodSandboxStatsRequest {
+    // Filter for the list request.
+    PodSandboxStatsFilter filter = 1;
+}
+
+message ListPodSandboxStatsResponse {
+    // Stats of the pod.
+    repeated PodSandboxStats stats = 1;
+}
+
+// PodSandboxAttributes provides basic information of the container.
+message PodSandboxAttributes {
```

Comment:
do we really need this message? it seems like more or less dup of existing `message PodSandbox`? Can we just use that instead?

https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1/api.proto#L523-L541

Reply:
yeah there's some extra info in PodSandbox, plus I'd like to stay consistent with container